src/main/java/hudson/plugins/nunit/NUnitArchiver.java

```diff
@@ -47,6 +48,16 @@ public NUnitArchiver(String root, TaskListener listener, String testResultsPatte
         this.testResultsPattern = testResultsPattern;
         this.unitReportTransformer = unitReportTransformer;
         this.failIfNoResults = failIfNoResults;
+        this.warningNUnitTestResultHanlingBehavior = "skipped";
```

Comment:
You probably meant han**d**ling
```suggestion
        this.warningNUnitTestResultHandlingBehavior = "skipped";
```